### ISSUE TITLE
Fast Pass failing : Ensures ARIA attributes are allowed for an element's role in fabric multi select dropdown.

### ISSUE THREAD

**Issue by vaishnavi-115** [microsoft/fluentui#19518]
Environment Information
Package version(s): "@fluentui/react": "^7.126.0",
Browser and OS versions: Chrome and Windows 10
Describe the issue:
Fast Paas is failing with error "Ensures ARIA attributes are allowed for an element's role" in fabric multi select dropdown when any item selected.

Please provide a reproduction of the issue in a codepen:
Actual behavior:
Fast Paas is failing with error "Ensures ARIA attributes are allowed for an element's role" in fabric multi select dropdown when any item selected.

Expected behavior:
Fast Paas should pass when any item selected in multi select dropdown control.

![image](https://user-images.githubusercontent.com/89511882/130780803-93812f70-c365-4ac0-b01c-7a9d4fd498e6.png)
![image](https://user-images.githubusercontent.com/89511882/130780890-63599b32-7081-4c43-8fc5-3db37ac83c05.png)

**Comment by gouttierre** [comment]
@vaishnavi-115 - Thanks for filing this issue with us. Our team will have a look but to set expectations, we’re likely not going to get to it immediately due to the fact we’re currently focused on larger coherence work items.

@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by khmakoto** [comment]
Hi @vaishnavi-115, I just ran the fast pass in Accessibility Checker in our v7 docs site with the multiselect `Dropdown` with a couple of options selected and I don't see the error you are describing above. Can you check by upgrading to the latest version if this is fixed? If not, could you provide a minimum codepen or codesandbox that reproduces the issue with detailed repro steps?
